### ISSUE TITLE
Issue after upgrading to 1.3.1

### ISSUE THREAD

**Issue by diego-paired** [Iterable/react-native-sdk#356]
Immediately after upgrading from 1.3.0 to 1.3.1, the following error prevents the app from booting:
```
TypeError: Cannot read property 'prototype' of undefined
    at ?anon_0_ (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=io.betterhalf.app:151379:101)
```

```
System:
    OS: macOS 12.3.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 108.48 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.14.0 - ~/.nvm/versions/node/v16.14.0/bin/node
    Yarn: 1.22.17 - /opt/homebrew/bin/yarn
    npm: 8.3.1 - ~/.nvm/versions/node/v16.14.0/bin/npm
    Watchman: 2021.09.27.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: Not Found
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 25, 27, 28, 29, 30, 31, 32
      Build Tools: 30.0.2, 30.0.3, 31.0.0, 32.0.0, 32.0.0, 33.0.0
      System Images: android-31 | Google APIs ARM 64 v8a, android-32 | Google APIs ARM 64 v8a, android-Sv2 | Google APIs ARM 64 v8a
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.1 AI-211.7628.21.2111.8309675
    Xcode: 13.3.1/13E500a - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.13 - /Users/diegolopez/.sdkman/candidates/java/current/bin/javac
  npmPackages:
    @react-native-community/cli: ^7.0.1 => 7.0.3
    react: ^17.0.2 => 17.0.2
    react-native: ^0.68.2 => 0.68.2
```

**Comment by alex-a-pereira** [comment]
I just upgraded to 1.3.1 last night and I might be hitting the same problem? we're using TS and the compiler is yelling at me - when I look at the type declarations, Iterable/IterableConfig are not exported. It also appears that those classes are not exported in index.ts in the source code https://github.com/Iterable/react-native-sdk/blob/master/ts/index.ts

![image](https://user-images.githubusercontent.com/36137079/168286011-df25b1e7-ce5b-43a3-b6a4-31831d22a4c8.png)

**Comment by huyhai** [comment]
same here

**Comment by alex-a-pereira** [comment]
@huyhai my PR has been merged to master so you can update your package.json so use the git URL and it will have the fix applied. https://stackoverflow.com/questions/16350673/depend-on-a-branch-or-tag-using-a-git-url-in-a-package-json

I'm not 100% sure that my PR addresses every issue with the SDK though

**Comment by roninopf** [comment]
@diego-paired Sorry for the delay in addressing this, we believe this is fixed in 1.3.2 thanks to @alex-a-pereira's PR.

If you run into a problem after that, please feel free to contact your CSM or to open another issue.
